tox.ini

```diff
@@ -4,52 +4,34 @@ DJANGO_SETTINGS_MODULE = tests.settings
 
 [tox]
 envlist =
-    py{37,38}-dj{22}-drf{38,39,310,311}-{pytest,mypy}
-    py{37,38}-dj{30}-drf{310,311}-{pytest,mypy}
-    py{37,38,39,310,311}-dj{31,32}-drf{311,312,313,314}-{pytest,mypy}
-    py{38,39,310,311}-dj{40,41}-drf{313,314}-{pytest,mypy}
-    py{38,39,310,311}-dj{42}-drf{314}-{pytest,mypy}
+    py{38,39,310,311,312}-dj42-drf{314,315}-pytest
```

Comment:
@browniebroke Why do you remove mypy checks here and in the next line?

Reply:
My understanding was that [DRF stubs](https://github.com/typeddjango/djangorestframework-stubs) (and [django-stubs](https://github.com/typeddjango/django-stubs)) are now linked to a specific version of DRF and Django. So you should ideally have stubs version == DRF version. This was changed in https://github.com/typeddjango/djangorestframework-stubs/releases/tag/3.14.0 and https://github.com/typeddjango/django-stubs/releases/tag/4.2.0

When you install `djangorestframework-stubs==3.15.*`, it installs [`django-stubs==5.0.*`](https://github.com/typeddjango/djangorestframework-stubs/blob/76a5d4f00948d65a3c588de81340d3c298aec4dc/setup.py#L23), which are for DRF 3.15 and Django 5.0 respectively. Although I just noticed that django-stubs=5.0.x has "partial" support for Django 4.2...

I suppose the problem with running mypy against the wrong versions is that the type check won't find type issues if the stubs are for a different version? What I mean is that if a Django API changed its signature between 4.2 and 5.0, but the stubs used by mypy are for v5, mypy won't detect the problem with 4.2.

Reply:
I can try to add them back if you still want them?